index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Weather App</title>
+	<link rel="stylesheet" href="src/style.css">
+	<link href="https://fonts.googleapis.com/css2?family=PT+Serif&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+		integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+	<script src="https://kit.fontawesome.com/200b8902d2.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+	<div class="boxed">
+		<div class="container">
+			<div class="topnav">
+				<a class="active" href="#home">Search</a>
+				<input type="text" placeholder="Enter Current Location">
+			</div>
+			<button type="button" class="btn btn-primary">Search for more locations here</button>
+			<h1>
+				San Jose
+			</h1>
+			<h2>
+				7/18/20 8pm
+			</h2>
+			<h3>
+				<i class="fas fa-cloud"></i>
+			</h3>
+			<h4>
+				75° F
+			</h4>
+			<div class="row">
```

Comment:
Good job in organizing this! It makes a lot of sense that each day is grouped together in it's own div!